content/docs/secure/encryption/deployment.md

```diff
@@ -16,6 +16,12 @@ the rest of the deployment process is described in the correspondent [CSI driver
 Hashicorp Vault must be [pre-configured](../vault) to support Encryption. The Vault server's IP address and port must be accessible 
 from the Kubernetes cluster where the CSI driver is to be deployed.
 
+## Rekey Controller
+
+The CSM Encryption Rekey Controller must be installed in order to be able to perform key rotation on encrypted volumes in a 
+Kubernetes cluster. The Rekey Controller can be installed via the standard Dell Helm Chart repository available at  https://github.com/dell/helm-charts.

```

Comment:
Please remove once extra space in front of the url.

Reply:
done